quinn-h3/src/qpack/prefix_string/encode.rs

```diff
@@ -93,6 +75,43 @@ impl HuffmanEncoder {
     }
 }
 
+// Write bits from `value` to the `out` slice
+//
+// Write the least significant `pos.count` bits from `value` to the position specified by
+// `(pos.byte, pos.bit)`. Writes may span multiple bytes. `out` is expected to be long enough
+// to write these bits; this is ensured by `HuffmanEncoder::ensure_free_space()`, which is
+// always called prior to calling this function.
+//
+// The bits to be written to are expected to be set to 1 when calling this function. Similarly,
+// this function maintains the invariant that unused bits in the output bytes are set to 1.
```

Comment:
Using 1 for unwritten data rather than 0 seems pretty unconventional. Is there a reason?

Reply:
I think this is described [here](https://tools.ietf.org/html/rfc7541#appendix-B). Added a reference to this in the doc comment for `write_bits()`.